BootloaderCorePkg/Tools/CfgDataTool.py

```diff
@@ -116,6 +117,23 @@ def RsaSignFile(self, InFile, OutFile, IncDat=False, IncKey=False):
         open(OutFile, 'wb').write(Bins)
 
 
+    def GetPrivateKeyInfo (self):
```

Comment:
Shoudn't this be a generic function ?  Should we move it to common  ?

Reply:
will move it commonutility